docs/source_values.rst

```diff
@@ -89,25 +101,38 @@ used to feed values into the device. Prior to v1.5, the
 It is also possible to set an output device's :attr:`~SourceMixin.source` to
 another output device, to keep them matching:
 
+.. image:: images/source_values/matching_leds.*
+    :align: center
+
```

Comment:
Perhaps add a sentence explaining what the example does? i.e. pressing the button will simultaneously light both the red and green LEDs.

Reply:
Done